Makefile

```diff
@@ -139,18 +140,33 @@ ui:
 
 .PHONY: install
 install: check-dcos
-	$(DCOS_CMD) package install --yes marathon-lb;\
-	$(DCOS_CMD) package install --yes kubernetes --package-version=$(KUBERNETES_FRAMEWORK_VERSION) --options="$(PATH_TO_PACKAGE_OPTIONS)";\
+	@echo "Installing Mesosphere Kubernetes Engine..."
+	$(DCOS_CMD) package install --yes kubernetes --package-version="$(KUBERNETES_FRAMEWORK_VERSION)"
+	@echo "Waiting for Mesosphere Kubernetes Engine to be up..."
+	@while [[ ! $$($(DCOS_CMD) kubernetes manager plan show deploy 2> /dev/null | head -n1 | grep COMPLETE ) ]]; do \
+		sleep 1; \
+	done
+	@echo "Creating a Kubernetes cluster..."
+	$(DCOS_CMD) kubernetes cluster create --yes --options="$(PATH_TO_PACKAGE_OPTIONS)"
+
+.PHONY: marathon-lb
+marathon-lb:
+	$(DCOS_CMD) package install --yes marathon-lb
+	@sleep 30
```

Comment:
Do we really need this sleep ?

Reply:
Maybe not. I'll check back and remove it if possible.

Reply:
👍